Updated guides and templates for Flight Compute environments

vlj91 opened the pull request:
- Updated some of the wording to match the displayed names in `alces template`
- Lowered the default resource requirement for running the `alces template` jobs to suit Cloud environments
- The output file directory creation in each of the templates would fail, so I've removed - as each of the templates already have their output file set through the SGE directive

Commit:
- Lowered the resource requirement for running template jobs
- Removed output dir creation as it failed, already writing to output file through SGE directive
- Remove the echo `OUTPUT_PATH` from templates

Commit:
Update displayed template names to match those displayed in `alces template`

mjtko commented:
As discussed, the failure was because the existence of the job script in the current working directory was clashing with the `JOB_NAME` (as, by default, it is the same) which meant a directory couldn't be created where the script wanted one.

Can you add the output directory back but change it so it makes it like this please:

`OUTPUT_PATH="$(pwd)/${JOB_NAME}-outputs/${JOB_ID}"`

Ta!

Commit:
Added output dir creation back to templates
As mentioned in: https://github.com/alces-software/clusterware-services/pull/14

mjtko commented:
Perfect, thanks!
